extensions/omni_sql/omni_sql.c

```diff
@@ -149,26 +149,58 @@ Datum raw_statements(PG_FUNCTION_ARGS) {
   rsinfo->setResult = tupstore;
 
   char *statement = PG_GETARG_CSTRING(0);
+  bool preserve_transactions = PG_GETARG_BOOL(1);
```

Comment:
`preserve_statement` should be checked for nullity.

And, as a matter of fact, so should be`statement` (my fault!)

Reply:
`statement` check was there, I missed it too. Moved all these checks together to the top for the sake of readability